src/includes/set-user/apple.mdx

```diff
@@ -9,3 +9,6 @@ SentryUser *user = [[SentryUser alloc] init];
 user.email = @"john.doe@example.com";
 [SentrySDK setUser:user];
 ```
+
+Since 6.0.1: If no user is set the SDK uses the installation id of the SDK as the
```

Comment:
```suggestion
Since 6.0.1: Sentry will by fallback to `installationId` if you do not provide a user identity.
```

Reply:
so it's similar to https://docs.sentry.io/platforms/android/enriching-events/identify-user/